tests/agent/test_rum.py

```diff
@@ -0,0 +1,13 @@
+from tests.endpoint import Endpoint
+from tests import utils
+import requests
+
+
+def test_rum(rum):
+    elasticsearch = rum.apm_server.elasticsearch
+    elasticsearch.clean()
+    endpoint = Endpoint(rum.url, "run_integration_test")
+
+    r = requests.get(endpoint.url)
+    assert r.status_code == 200
+    utils.check_elasticsearch_transaction(elasticsearch, 1, {'query': {'term': {'processor.event': 'transaction'}}})
```

Comment:
Is the number of spans generated here unpredictable?  If not, you could avoid having a custom query here.  I always get 7 docs locally.

Reply:
Unfortunately, the number of spans is not predictable. The reason is that we generate some of the spans based on the browser navigation timing and depending how fast or slow the test runs, the duration of some of the spans might be zero which we remove before generating the final payload.